README.rst

```diff
@@ -156,6 +159,21 @@ Note: Must install docker desktop.
     $ poetry install
     $ poetry run pytest -v --splunk-version=${splunk-version} -m docker -m ${test-marker} tests/e2e
 
+Troubleshooting:
+
+1. If you face an error like this 'argparse.ArgumentError: argument -K/--keepalive: conflicting option strings: -K, --keepalive'
+
+    This is likely to happen if you have older version of PSA requriements install, to solve this try to uninstall lovely-pytest-docker and pull the latest main branch
+
+2. If while running the tests you face error as below
+
+.. code:: bash
```

Comment:
The error message is not displayed in the preview.

Reply:
Updated